docs/concepts/training-data.mdx

```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Data bundles
+# Training
 
-A _data bundle_ in Cradl is a model-specific collection of one or more [Datasets](datasets.mdx). Before training a model, you must specify a data bundle that the model will use as training data. Since the quality of the trained model depends on the quality of the training data, we require that any data bundles used for training must pass a statistical test in the form of a [Data Report](training-data.mdx#data-report) before training can begin.
+A _training_ in Cradl is the process of training a machine learning model on the data you provide.
+It can be summed up in 3 simple steps:
+1. User requests training in the Cradl app, or with one of our SDKs or CLI.
+2. Our team of Data Scientists verify that the training is feasible, and starts the training.
+3. User receives a new model that has been trained according to the request.
 
-## Creating a data bundle
+:::info
+We make sure to always provide a model whose architecture is based on the latest research to make sure that our customers get the most out of their data.
+:::
+
+## Request a training
+
+The simplest way to request a training is to use the training wizard in the Cradl app, just go to [**Models**](https://app.cradl.ai/models) > **Your model** > **Training jobs** > **New training**.
+If you want to request a training with one our SDKs or CLI you need to create a [**Data Bundle**](training-data.mdx#data-bundle) first.
+
+<Tabs groupId="programming-language">
+<TabItem value="cli" label="CLI" default>
+
+```bash
+las models create-training --name "Initial training of receipt model" <modelId> <dataBundleId>
+```
+
+</TabItem>
+<TabItem value="curl" label="cURL">
+
+```bash
+curl -X POST 'https://api.lucidtech.ai/v1/models/<modelId>/trainings' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer eyJra...' \
+--data-raw '{
+    "modelId": "<modelId>",
+    "dataBundleIds": [
+        "<dataBundleId>"
+    ],
+    "name": "Initial training of receipt model"
+}'
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+data_bundle = client.create_training(model_id='<modelId>', data_bundle_ids=['<dataBundleId>'], name='Initial training of receipt model')
+```
+
+</TabItem>
+</Tabs>
+
+After the training is requested you will receive an email that verifies the request, and when the training is completed.
+You can also check the request of your training, by looking at its status.
+
+
+## Data Bundle
+
+A _data bundle_ in Cradl is a model-specific collection of one or more [Datasets](datasets.mdx).
+Before training a model, you must specify a data bundle that the model will use as training data.
+Since the quality of the trained model depends on the quality of the training data,
+we require that any data bundles used for training must pass a statistical test in the form of a [Data Report](training-data.mdx#data-report) before training can begin.
```

Comment:
What are the requirements for a statistical test to pass?

Reply:
They are often subject to change, so putting down something specific here would quickly be outdated I am afraid.